### ISSUE TITLE
NordVPN 3.18.4 fails to resolve google.com or any domain really

### ISSUE THREAD

**Issue by mrpainte** [NordSecurity/nordvpn-linux#592]
`pi@rp1:~ $ nordvpn status
Status: Connected
Server: Mexico #<server number>
Hostname: mx<servernumber>.nordvpn.com
IP: <ip of server>
Country: Mexico
City: Mexico
Current technology: NORDLYNX
Current protocol: UDP
Transfer: 2.99 GiB received, 1.52 GiB sent
Uptime: 55 minutes 10 seconds
pi@rp1:~ $ nordvpn settings
Technology: NORDLYNX
Firewall: enabled
Firewall Mark: 0xe1f1
Routing: enabled
Analytics: enabled
Kill Switch: disabled
Threat Protection Lite: disabled
Notify: enabled
Tray: enabled
Auto-connect: enabled
IPv6: disabled
Meshnet: disabled
DNS: 1.1.1.1
LAN Discovery: disabled
Virtual Location: enabled
Allowlisted ports:
           22 (UDP|TCP)
         8081 (UDP|TCP)
Allowlisted subnets:
        192.168.0.0/16
pi@rp1:~ $ nordvpn -v
NordVPN Version 3.18.4
`

these are my settings above for nordvpn. the next section is for showing connection to nord, curling google, disconnecting from nord and attempting to curl again.

`pi@rp1:~ $ nordvpn connect mexico
Connecting to Mexico #106 (mx106.nordvpn.com)
You are connected to Mexico #106 (mx106.nordvpn.com)!
pi@rp1:~ $ curl google.com
curl: (6) Could not resolve host: google.com
pi@rp1:~ $ nordvpn d
You are disconnected from NordVPN.
How would you rate your connection quality on a scale from 1 (poor) to 5 (excellent)? Type 'nordvpn rate [1-5]'.
-pi@rp1:~ $ curl google.com
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="http://www.google.com/">here</A>.
</BODY></HTML>
`
Reference, i was recently using NordVPN 3.17.0 and did not have these issues. might have to go back to that version to be able to use nordvpn fully. Currently using 3.18.4 which is the latest version for linux.

**Comment by mrpainte** [comment]
even with disabling DNS to use nordvpn dns i cant get to google or any website via domains.

**Comment by mrpainte** [comment]
can you give me direction on how to obtain the logs? checking /var/log/nordvpn/daemon.log does not seem to be the true log file

**Comment by mrpainte** [comment]
Sep 03 09:05:24 rp1 nordvpnd[116106]: 2024/09/03 09:05:24 [NC] failed to connect:  network Error : dial tcp: lookup nc-unit6-mqtt.nordvpn.com on 192.168.10.1:53: write udp 192.168.10.23:39774->192.168.10.1:53: write: operation not permitted

i do see this log however (last line).. with nordvpn enabled, it seems DNS is trying to access my routers DNS instead of nordvpn's DNS? i find that odd

**Comment by keliramu** [comment]
Ok, I got it, problem is that app is trying to use your router as a DNS server.

**Comment by mrpainte** [comment]
yeah. which i know works since when  im off the VPN it works but when on the VPN it fails. i did install 3.18.2 and it is working but 3.18.4 is not

**Comment by MatthewGreen** [comment]
@keliramu I'm seeing this issue in 3.19.1 as well. Times are in EDT. 192.168.1.57 is my local adguard box, IPv4 of the device using nordvpn is 192.168.1.49. Running Ubuntu 22.04.

## Settings
```
Technology: NORDLYNX
Firewall: enabled
Firewall Mark: 0xe1f1
Routing: enabled
Analytics: disabled
Kill Switch: enabled
Threat Protection Lite: disabled
Notify: disabled
Tray: disabled
Auto-connect: enabled
IPv6: disabled
Meshnet: disabled
DNS: disabled
LAN Discovery: disabled
Virtual Location: enabled
Post-quantum VPN: disabled
Allowlisted subnets:
        192.168.1.1/24
        192.168.4.1/24
```

## Logs
```
Nov 27 10:25:33 HOSTNAME nordvpnd[112]: Error: dial tcp: lookup downloads.nordcdn.com on 192.168.1.57:53: write udp 192.168.1.49:33124->192.168.1.57:53: write: operation not permitted
Nov 27 10:42:58 HOSTNAME nordvpnd[112]: 2024/11/27 10:42:58 [NC] failed to connect:  network Error : dial tcp: lookup nc-unit5-mqtt.nordvpn.com on 192.168.1.57:53: write udp 192.168.1.49:42711->192.168.1.57:53: write: operation not permitted
Nov 27 10:47:27 HOSTNAME nordvpnd[112]: 2024/11/27 10:47:27 [NC] failed to connect:  network Error : dial tcp: lookup nc-unit5-mqtt.nordvpn.com on 192.168.1.57:53: write udp 192.168.1.49:37748->192.168.1.57:53: write: operation not permitted
```

**Comment by klondikemarlen** [comment]
Some recent update has broken `traceroute` to google.com.
i.e.
```
$ traceroute google.com
traceroute to google.com (192.0.0.88), 30 hops max, 60 byte packets
 1  * * *
 2  * * *
 3  * * *
...
30  * * *
```

Note that nordvpn has also aliased the google.com IP to a local IP.

Setting nordvpn DNS to CloudFlare via
```
nordvpn set dns 1.1.1.1 1.0.0.1
```
seems to have fixed the issue for me, though connectivity is slow.
